k8s/charts/openebs/templates/deployment-maya-apiserver.yaml

```diff
@@ -52,6 +52,8 @@ spec:
           value: "{{ .Values.apiserver.sparse.enabled }}"
         - name: OPENEBS_IO_CSTOR_POOL_SPARSE_DIR
           value: "{{ .Values.ndm.sparse.path }}"
+        - name: OPENEBS_IO_CREATE_DEFAULT_STORAGE_CONFIG
+          value: "{{ .Values.defaultConfigs.storageConfig.enabled }}"
```

Comment:
Call this `defaultStorageConfig`. This should get included in the values.yaml. There should also be an entry in this yaml to add `OPENEBS_IO_CSTOR_TARGET_DIR ` ENV pointing to `.Values.ndm.sparse.path`

`defaultStorageConfig.enabled` should be documented in readme.

Reply:
Make sure the ENV variable are available in the operator.yaml also: 
- OPENEBS_IO_CSTOR_POOL_SPARSE_DIR
- OPENEBS_IO_CSTOR_TARGET_DIR
- OPENEBS_IO_CREATE_DEFAULT_STORAGE_CONFIG

Reply:
Added the ENV variables. PTAL